caput/_version.py

```diff
@@ -116,7 +116,7 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
     """
     rootdirs = []
 
-    for _ in range(3):
```

Comment:
`i` is still not used here, so this will trigger some linter warnings.

Reply:
I saw that! I want to ignore the linter warnings for this file, but would you prefer for me to modify `_version.py`?